.storybook/mdx-code-block-rewrite.js

```diff
@@ -19,7 +19,7 @@ module.exports = function rewriteExampleCodeBlock(source) {
   if (!hasArgsTable) imports.push('ArgsTable');
   return (
     (imports.length && hasSpecialBlocks
-      ? `import {${imports.join(',')}} from '@storybook/addon-docs/blocks';\n\n`
+      ? `import {${imports.join(',')}} from '@storybook/addon-docs';\n\n`
```

Comment:
All changes from `@storybook/addon-docs/blocks` to `@storybook/addon-docs` is because of a deprecation added to 6.3 that isn't transpiled to ES5 which fails in IE11. The "fix" is to change imports. https://github.com/storybookjs/storybook/issues/15315